### PR TITLE
Add guide on how to build, unit test and system test Vespa on CentOS …

### PR DESCRIPTION
**Pull request by geirst**
…7 using vespa-dev-centos7.

@toregge and @bjorncs please review.
@aressem @vekterli @baldersheim @havardpe @arnej27959 @gjoranv FYI

### CODE REVIEW

**Comment by aressem** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,10 +1,191 @@
...
+
+    mkdir -p $HOME/git
+    cd $HOME/git
+    git clone git@github.com:vespa-engine/vespa.git
+
+##### Build Java modules
+
+    cd $HOME/git/vespa
+    ./bootstrap.sh full
+    mvn clean install --threads 1C -DskipJavadoc -DskipTests
```
[the remark this answers is not in the document]
Comment (geirst) [reply README.md:135]: According to @bjorncs we don't distribute public snapshot artifacts so `--no-snapshot-updates` should not be necessary. @bjorncs what do you think of `--batch-mode`?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,191 @@
...
+    ./bootstrap-cmake.sh -u .
+    make -j 9
+
+Set the number of compilation threads (-j argument) to the number of CPU cores + 1.
+
+##### Install modules
+
+    make install/fast
+
+Default install directory is $HOME/vespa (VESPA_HOME).
```
[the remark this answers is not in the document]
Comment (geirst) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,10 +1,191 @@
...
+    nodeserver.sh
+
+##### Run system test in another terminal
+
+    cd $HOME/git/system-test/tests/search/basicsearch
+    runtest.sh basic_search.rb
+
+Note that if you run a system test that compiles a Java bundle the Vespa version must be specified as well:
+
+    runtest.sh basic_search.rb --vespa-version=7-SNAPSHOT
```
[the remark this answers is not in the document]
Comment (geirst) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -1,10 +1,191 @@
...
+##### Checkout Vespa repo
+
+    mkdir -p $HOME/git
+    cd $HOME/git
+    git clone git@github.com:vespa-engine/vespa.git
+
+##### Build Java modules
+
+    cd $HOME/git/vespa
+    ./bootstrap.sh full
```
[the remark this answers is not in the document]
Comment (geirst) [reply README.md]: Fixed

File: README.md [hunk 5]
```diff
@@ -1,10 +1,191 @@
...
+
+This adds yourself as user in the container, copies authorized keys to ensure ssh can be used,
+and sets environment variables needed for building Vespa.
+
+#### Build the vespa-dev-centos7 Docker image (optional)
+
+    cd $HOME/git/docker-image-dev/dev/centos7
+    docker build -t vespaengine/vespa-dev-centos7:latest .
+
+Use this for testing or if the Docker image (for any reason) is not available from Docker Hub.
```
[the remark this answers is not in the document]
Comment (geirst) [reply README.md]: Fixed

**Comment by geirst** [comment]
@toregge PTAL

**Pull request merged**